default/python/universe_generation/empires.py

```diff
@@ -113,18 +113,24 @@ def find_home_systems_for_min_jump_distance(self, systems_pool, min_jumps):
             if system not in self.system_merit:
                 self.system_merit[system] = calculate_home_system_merit(system)
 
-        best_case_worst_merit = sorted(self.system_merit.values())[self.num_home_systems - 1]
-        current_worst_merit = 0
+        # In the list of systems sorted by merit, if the first num_home_systems are all min_jumps
+        # apart, then the merit of the num_home_systems nth system is the
+        # best_case_merit_lower_bound that can be achieved.
+        best_case_merit_lower_bound = sorted(self.system_merit.values())[self.num_home_systems - 1]
```

Comment:
Ok, that makes sense. I'd probably go with the use of "score"/"rating" to avoid the confusion, that approach sounds like the most reasonable to me.

Moot point though, as `best_case_merit_lower_bound` got thrown out in the meantime...